aea/cli/issue_certificates.py

```diff
@@ -48,15 +50,37 @@ def issue_certificates(click_context):
 
 def issue_certificates_(ctx: Context):
     """Issue certificates for connections that require them."""
-    agent_config_manager = AgentConfigManager.load(ctx.cwd)
-
     # agent_config_manager.
-    for connection_id in agent_config_manager.agent_config.connections:
-        _process_connection(ctx, agent_config_manager, connection_id)
+    for connection_id in ctx.agent_config.connections:
+        cert_requests = _get_cert_requests(ctx, connection_id)
+        _process_connection(ctx, cert_requests, connection_id)
 
     click.echo("All certificates have been issued.")
 
 
+def _get_cert_requests(ctx: Context, connection_id: PublicId) -> List[CertRequest]:
+    """
+    Get certificate requests, taking the overrides into account.
+
+    :param ctx: the CLI context.
+    :param connection_id: the connection id.
+    :return: the list of cert requests.
+    """
+    component_id = ComponentId(ComponentType.CONNECTION, connection_id)
+    overrides = ctx.agent_config.component_configurations.get(component_id, {})
```

Comment:
So we cannot use AgentConfigManager? That's all becoming very messy and confusing...